index.js

```diff
@@ -2,7 +2,7 @@ const events = require('events')
 const through2 = require('through2')
 const inherits = require('inherits')
 
-function SignalhubWs (app, urls) {
+function SignalhubWs (app, urls, wsClass = window.WebSocket) {
```

Comment:
May we declare `const WebSocket = window && window.WebSocket ? window.WebSocket : null`. This way we avoid errors when window is undefined.

Then `(app, urls, ws = WebSocket)` 

And if no implementation is given (ie `ws === null`, let's do `throw TypeError('')`?